testkit/mocksessionmanager.go

```diff
@@ -35,6 +36,8 @@ type MockSessionManager struct {
 	Dom     *domain.Domain
 	Conn    map[uint64]session.Session
 	mu      sync.Mutex
+
+	internalSessions map[interface{}]struct{}
```

Comment:
The session used by the internal transaction of DDL is not added to the session manager. And you fix it, but where does DDL call the `MockSessionManager`?